src/javascript/app/pages/trade/update_values.js

```diff
@@ -14,24 +14,24 @@ const updatePurchaseStatus = (final_price, pnl, profit, localized_contract_statu
     $cost.html($('<div/>', { text: localize('Final price') }).append($('<p/>', { html: formatMoney(currency, final_price) })));
     dataManager.setPurchase({
         pr_heading         : localized_contract_status,
-        pr_tablePayout     : localize('Buy price') ,
-        pr_tablePayoutValue: formatMoney(currency, Math.abs(pnl)),
-        pr_tableCost       : localize('Final price'),
-        pr_tableCostValue  : formatMoney(currency, final_price),
+        pr_table_payout     : localize('Buy price') ,
+        pr_table_payout_value: formatMoney(currency, Math.abs(pnl)),
+        pr_table_cost       : localize('Final price'),
+        pr_table_cost_value  : formatMoney(currency, final_price),
```

Comment:
it looks like GH actions are complaining about alignment 'Missing space after key'